docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,171 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives an input stream of POJOs and inserts them as rows in a database table in a fault-tolerant way.
+
+## Overview
+The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of each window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by `batchSize` property. The tuples in a window are stored in a check-pointed collection which is cleared in the `endWindow()`. The operator writes a tuple exactly once in the database.
+This operator internally extends the functionality of `AbstractPassThruTransactionableStoreOutputOperator`. AbstractPassThruTransactionableStoreOutputOperator is the base implementation for a pass-through output adapter of a transactional store. AbstractPassThruTransactionableStoreOutputOperator implementation provides the "exactly once" guarantee. "Pass-through" means it does not wait for end window to write to the store. It will begin transaction at `beginWindow` and write to the store as the tuples come and commit the transaction at `endWindow`.
+
+**Note**: For enforcing exactly once semantics a table named 'dt_meta' must exist in the database. The sample SQL to create the same is as follows
+```
+"CREATE TABLE IF NOT EXISTS dt_meta (dt_app_id VARCHAR(100) NOT NULL, dt_operator_id INT NOT NULL, dt_window BIGINT NOT NULL, UNIQUE(dt_app_id,dt_operator_id,dt_window))".
+```
+**Note**: Additionally this operator assumes that the underlying database/table in which records are to be added supports transactions. If the database/table does not support transactions then a tuple may be inserted in a table more than once in case of auto recovery from a failure (violation of exactly once semantics).
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***0.9.4***
+3. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcTransactionableOutputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcTransactionableOutputOperator.html)***
+
+## How to Use?
+ Concrete subclasses need to choose an appropriate class (such as String or an appropriate concrete java class). Also implement a couple of abstract methods (if not using AbstractJdbcPOJOOutputOperator): `setStatementParameters(PreparedStatement statement, T tuple)` to set the parameter of the insert/update statement (which is a PreparedStatement) with values from the tuple and `getUpdateCommand()` the sql statement to update a tuple in the database. Note that these methods are also defined in AbstractJdbcPOJOOutputOperator.
```

Comment:
I have mentioned "if not using AbstractJdbcPOJOOutputOperator" because the user may choose to directly use the base class (for e.g. as done here https://www.datatorrent.com/blog/end-to-end-exactly-once-with-apache-apex/) 

Changed sql to SQL